### ISSUE TITLE
Issues: `remark-grid-tables` I18n alignment error.

### ISSUE THREAD

**Issue by cangSDARM** [zestedesavoir/zmarkdown#391]
Here is the source code

```
+-------+----------+------+
|   语法的联系&区别         |
+-------+----------+------+
| Sub   | Headings | Too  |
+=======+==========+======+
| cell  | column spanning |
+ spans +----------+------+
| rows  | normal   | cell |
+-------+----------+------+
| multi | cells can be    |
| line  | *formatted*     |
|       | **paragraphs**  |
| cells |                 |
| too   |                 |
+-------+-----------------+
```

As you can see, the Chinese line(Heading line), misalignment.
And it rendered like this:

![截屏2020-04-14 15 01 54](https://user-images.githubusercontent.com/20441896/79195471-ea3c4500-7e60-11ea-827f-13ad784eab2f.png)

For right view, lt should like this:
![截屏2020-04-14 15 02 53](https://user-images.githubusercontent.com/20441896/79195550-093ad700-7e61-11ea-94c3-907526e17f73.png)

**Comment by vhf** [comment]
It should definitely work, you might have to upgrade `remark-grid-tables` though. 

Did you try here: https://zestedesavoir.github.io/zmarkdown/ ?

**Comment by arobase-che** [comment]
Didn't #370 solved that ?

**Comment by TitiAlone** [comment]
I agree, it should have been solved, could you please upgrade to version `2.1.0` and see if it works?

Maybe we still have problems with that, but I'd say it's rather unlikely.

**Comment by cangSDARM** [comment]
It seems that it is indeed a problem with my version. Thanks!
